docs/docs/manual/running.md

```diff
@@ -286,13 +293,16 @@ Locate the `refine.ini` file, and add `JAVA_OPTIONS=` before the `-Drefine.prefe
 JAVA_OPTIONS=-Drefine.autosave=2
 JAVA_OPTIONS=-Drefine.port=3334
 JAVA_OPTIONS=-Drefine.data_dir=usr/lib/OpenRefineWorkspace
+JAVA_OPTIONS=-Dhttps.proxyHost=yourproxyhost
+JAVA_OPTIONS=--Dhttp.proxyHost=yourproxyhost
```

Comment:
Double `-` here

Reply:
Doing it right away

Reply:
The double `-` is still here.

Reply:
Oh I overlooked that. Updating it in a few